packages/integrations/netlify/src/index.ts

```diff
@@ -596,6 +597,7 @@ export default function netlifyIntegration(
 					},
 					session,
 					vite: {
+						plugins: [netlifyVitePlugin()],
```

Comment:
Shouldn't we add this plugin only when running the `dev` command? That's what I understood from the PR description

Reply:
It looks like the plugin only implements the `configureServer` hook, so it won't affect builds: https://github.com/netlify/primitives/blob/main/packages/vite-plugin/src/main.ts

Reply:
I'm happy to make this more explicit on the adapter side if needed. What's your recommendation?